hi-ml-cpath/testhisto/testhisto/utils/test_plots_utils.py

```diff
@@ -22,16 +22,17 @@ def test_plots_handler_wrong_class_names() -> None:
     assert "No class_names were provided while activating confusion matrix plotting." in str(ex)
 
 
-def test_plots_handler_slide_thumbnails_without_slide_dataset() -> None:
+@pytest.mark.parametrize("slide_plot_option", [PlotOption.SLIDE_THUMBNAIL, PlotOption.ATTENTION_HEATMAP])
+def test_plots_handler_slide_plot_options_without_slide_dataset(slide_plot_option: PlotOption) -> None:
     with pytest.raises(ValueError) as ex:
         container = MockDeepSMILETilesPanda(tmp_path=Path("foo"))
         container.setup()
         container.data_module = MagicMock()
         container.data_module.train_dataset.n_classes = 6
         outputs_handler = container.get_outputs_handler()
-        outputs_handler.test_plots_handler.plot_options = {PlotOption.SLIDE_THUMBNAIL_HEATMAP}
+        outputs_handler.test_plots_handler.plot_options = {slide_plot_option}
         outputs_handler.set_slides_dataset_for_plots_handlers(container.get_slides_dataset())
-    assert "You can not plot slide thumbnails and heatmaps without setting a slides_dataset." in str(ex)
+    assert f"Plot option {slide_plot_option} requires a slides dataset" in str(ex)
```

Comment:
[Minor] You can achieve this directly by passing `pytest.raises(ValueError, match="...")` ([docs](https://docs.pytest.org/en/7.1.x/how-to/assert.html#assertions-about-expected-exceptions)).